extensions/mssql/src/mssql.d.ts

```diff
@@ -899,6 +899,7 @@ declare module 'mssql' {
 			Column = "Column",
 			Database = "Database",
 			DatabaseRole = "DatabaseRole",
+			DatabaseProperties = "DatabaseProperties",
```

Comment:
These should be the object types of the node - e.g. `Database` instead of `DatabaseProperties`